### PR TITLE
Flow control at the boundary of Akka Streams and external library

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: _posts/2016-09-05-flow-control-at-the-akka-stream-boundary.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Flow control at the boundary of Akka Streams and a data provider"
+description: ""
+author: Martynas Mickevičius
+category: integrations
+tags: [streams,connectors,integration]
+---
+{% include JB/setup %}
```
Comment (drewhk) [inline _posts/2016-09-05-flow-control-at-the-akka-stream-boundary.md:11]: we should link to earlier posts

Comment (ktoso) [reply _posts/2016-09-05-flow-control-at-the-akka-stream-boundary.md:11]: And from the old ones to the new ones too ⭕

**Comment by 2m** [comment]
Added series support by adding a new layout, that adds a list of all posts in the series, with the current one being pointed out. This is how it looks like on this new post:

series

**Comment by drewhk** [comment]
can we put this at the top instead of the bottom?

**Comment by 2m** [comment]
Yes, we can. But I thought they fit better at the bottom as it is a nice continuation when one finishes reading the post.

**Comment by drewhk** [comment]
_If_ one gets to the bottom :) Ok, we can tweak it later.

**Comment by ktoso** [comment]
Good idea on moving the links up, can be done after once we're done with the series too.

**Comment by ktoso** [comment]
LGTM

File: _includes/series [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    {% for p in series_posts %}
+        {% if p.url == page.url %}
+            <li class="active">{{p.weight}}  → {{p.title}}</li>
+        {% else %}
+            <li>
+                <a href="{{p.url | prepend: site.baseurl}}">{{p.weight}}  {{p.title}}</a>
+            </li>
+        {% endif %}
+    {% endfor %}
+</ol>
```
Comment (ktoso) [inline _includes/series:11]: cool

**Pull request merged**